src/ec/packages/ec-component-select/select.html.twig

```diff
@@ -0,0 +1,103 @@
+{#
+  Parameters:
+  - "id" (string) (default: '')
+  - "options" (array) (default: []):
+    - "value" (string) (default: '')
+    - "label" (string) (default: '')
+  - "disabled" (boolean) (default: false)
+  - "invalid" (boolean) (default: false)
+  - "name" (string) (default: '')
+  - "hide_label" (boolean) (default: '')
+  - "icon_path" (string) (default: ''): file containing the svg icons
+  - "extra_classes" (optional) (string) (default: '') Extra classes (space separated) for the select
+  - "extra_attributes" (optional) (array) (default: []) Extra attributes for select
+    - "name" (string) Attribute name, eg. 'data-test'
+    - "value" (string) Attribute value, eg: 'data-test-1'
+  Blocks:
+  - "helper_text"
+  - "invalid_text"
+  - "label"
+#}
+
+{# Internal properties #}
+{% set _id = id|default('') %}
+{% set _options = options|default([]) %}
+{% set _disabled = disabled|default(false) %}
+{% set _invalid = invalid|default(false) %}
+{% set _name = name|default('') %}
+{% set _hide_label = hide_label|default(false) %}
+{% set _icon_path = icon_path|default(false) %}
+
+{% set _helper_text = helper_text|default('') %}
+{% set _invalid_text = invalid_text|default('') %}
+{% set _label = label|default('') %}
+
+{% set _css_class = ['ecl-select'] %}
+{% if _invalid %}
+  {% set _css_class = _css_class|merge(['ecl-select--invalid']) %}
+{% endif %}
+{% set _css_class = _css_class|join(' ') %}
+
+{% set _extra_attributes = '' %}
+
+{# Internal logic - Process properties #}
+{% if extra_classes is defined and extra_classes is not empty %}
+  {% set _css_class = _css_class ~ ' ' ~ extra_classes %}
+{% endif %}
+
+{% if extra_attributes is defined and extra_attributes is not empty and extra_attributes is iterable %}
+  {% for attr in extra_attributes %}
+    {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value ~ '"' %}
+  {% endfor %}
+{% endif %}
+
+{# Print the result #}
+{% spaceless %}
+<div class="ecl-form-group ecl-form-group--select">
+  {%- if _label is not empty -%}
+    <label 
+      class="ecl-form-label{{- invalid ? ' ecl-form-label--invalid' -}}{{- hide_label ? ' ecl-form-label--hidden'-}}"
+      for={{- _id -}}
```

Comment:
what if `_id` contains a forbidden char?

> The attribute value can remain unquoted if it doesn't contain ASCII whitespace or any of " ' ` = < or >. Otherwise, it has to be quoted using either single or double quotes. 

— https://html.spec.whatwg.org/multipage/introduction.html#intro-early-example